### PR TITLE
fix(backend): use new backend for orgdiagram for valid results

### PR DESCRIPTION
**Pull request by paule96**

### CODE REVIEW

**Comment by paule96** [comment]
@mschwrdtnr this is more or less a big restyle. :D so maybe the label was good. The result of the restyle is that the bug is fixed yes, but the big thing is that we use the new backend for the organigram. So can I add the label again? :P

File: Phonebook.Frontend/src/app/services/api/organigram.service.ts [hunk 1]
```diff
@@ -97,3 +89,30 @@ export class UnitTreeNode {
     this.learners = learners;
   }
 }
+
+export class OrgUnit {
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/services/api/organigram.service.ts:93]:
```suggestion
class OrgUnit {
```

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/services/api/organigram.service.ts:93]: Don't make things available globally that we don't want to

**Comment by DanielHabenicht** [comment]
Why did this not trigger any build?

**Comment by paule96** [comment]
It does?

image

**Comment by mschwrdtnr** [comment]
> @mschwrdtnr this is more or less a big restyle. :D so maybe the label was good. The result of the restyle is that the bug is fixed yes, but the big thing is that we use the new backend for the organigram. So can I add the label again? :P

For my restyled means something different and your title says "fix" 🤔

**Comment by paule96** [comment]
Yes because I can't use both. Or can I? Can I do something like: fix(something) feature(something)

So restyle only includes formating?

**Comment by mschwrdtnr** [comment]
> Yes because I can't use both. Or can I? Can I do something like: fix(something) feature(something)
> 
> So restyle only includes formating?

I don't make the rules^^ We should agree on something and write it in our "Contribution Guidelines" @ #412

**Comment by DanielHabenicht** [comment]
> Yes because I can't use both. Or can I? Can I do something like: fix(something) feature(something)
> 
> So restyle only includes formating?

why should it be both? 
It should be either a fix or a new feature. If it is both it should be two different commits. 
`restyle` on restyles the code. It does not change the functionality.

**Comment by DanielHabenicht** [comment]
ready to merge? @paule96

**Comment by paule96** [comment]
> Your backend seems to have an error for development

have you an url for me?

**Comment by DanielHabenicht** [comment]
@paule96 https://pr-664.demo-phonebook.me/api/OrgUnit

**Comment by paule96** [comment]
lol don't has the problem local... what is different in the demo...

**Pull request merged**